pkg/kubelet/cm/topologymanager/policy_none_test.go

```diff
@@ -0,0 +1,68 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package topologymanager
+
+import (
+	"testing"
+)
+
+func TestName(t *testing.T) {
+	tcases := []struct {
+		name     string
+		expected string
+	}{
+		{
+			name:     "New None Policy",
+			expected: "none",
+		},
+	}
+	for _, tc := range tcases {
+		policy := NewNonePolicy()
+		if policy.Name() != tc.expected {
+			t.Errorf("Expected Policy Name to be %s, got %s", tc.expected, policy.Name())
+		}
+	}
+}
+
+func TestCanAdmitPodResult2(t *testing.T) {
```

Comment:
Why there is a 2 in the function name?

Reply:
Strict policy test and preferred policy test have similar functions named ''TestCanAdmitPodResult" and "TestCanAdmitPodResult1" respectively, so just following that naming convention.

Reply:
I think renaming it to TestPolicyNoneCanAdmitPodResult is better, and the other tests should be renamed respectively.

Reply:
Sure, I will update all three.